adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -570,14 +594,19 @@ func createHealthCheck() []*corev3.HealthCheck {
 	}
 }
 
-func createUpstreamTLSContext(upstreamCerts []byte, address *corev3.Address) *tlsv3.UpstreamTlsContext {
+func createUpstreamTLSContext(upstreamCerts []byte, address *corev3.Address, isRateLimitCluster bool) *tlsv3.UpstreamTlsContext {
```

Comment:
Please refer to https://github.com/wso2/product-microgateway/pull/3142/files#r1044083874